torchqdynamics/odeint.py

```diff
@@ -27,6 +51,35 @@ def forward(self, t: float, y: Tensor) -> Tensor:
         """
         pass
 
+    def __init__(self, H):
+        """
+        Args:
+            H: Hamiltonian, of shape `(b_H, n, n)`.
+        """
+        if isinstance(H, OperatorLike.__args__):
+            if H.ndim == 3:
+                H = H[:, None, ...]  # (b_H, 1, n, n)
+        self._H = H
+        self._previous_t = False
+        self._cache = {}
+
+    @H_dependent
+    def H(self, t):
+        if isinstance(self._H, OperatorLike.__args__):
+            return self._H
+        elif callable(self._H):
+            return self._H(t)
+        else:
+            raise Exception('Piece-wise constant Hamiltonians not supported yet')
+
+    def _H_changed(self, _t):
```

Comment:
```suggestion
    def _H_changed(self, t):
```

Reply:
This silences the IDE warnings because the variable is unused for now ;)